docs_src/microservices/application/BuiltIn.md

```diff
@@ -136,6 +136,15 @@ There is one Core Data function that enables interactions with the Core Data RES
     NewCoreDataSimpleReading("my-profile", "my-device", "my-resource", "string").PushToCoreData
     ```
 
+### Wrap Into Event
```

Comment:
Need to create new `## Event` section and copy table from `## Core Data` 

Also just noticed that `CoreData` in the factory function names. This should be replaced with `Event`. I.e. `NewEventSimpleReading`